docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -7,21 +7,24 @@ ms.localizationpriority: high
 
 # SharePoint Embedded Container Types
 
-In SharePoint Embedded, files and documents are stored in Containers, each identified by a Container Type, which is a property stamped on every instance. A Container Type links all Containers to a specific Azure Subscription for billing. A single SharePoint Embedded Application owns each Container Type, and each SharePoint Embedded Application can own only one Container Type. It specifies the access permissions a SharePoint Embedded Application has for all Containers of that type, such as creating, reading, writing, deleting containers, and managing container permissions.
+In SharePoint Embedded, files and documents are stored in Containers, with each container instance stamped with an immutable Container Type property. Each Container Type is owned by only one SharePoint Embedded application, and each SharePoint Embedded application can only own one Container Type. Container Type serves three general purposes:
 
-## Two Kinds of Container Types
+1.	It links all Containers of that type to a specific Azure Subscription for billing. 
+2.	It specifies the permissions a SharePoint Embedded Application has for all container instances of that type, such as creating, reading, writing, deleting containers and content, and managing container permissions.
+3.	It configures expected behaviors of all container instances of that type, such as sharing, content discoverability, etc.
 
-There are two kinds of Container Types that a developer tenant can create:
+Container Type is created and managed by the SharePoint Embedded application developer, and is strongly tied to the SharePoint Embedded application. Container Type can be created with or without linkage to an Azure billing profile, for trial/development and production purposes respectively.
 
-1. [Trial Container Types](#trial-container-types)
-1. [Standard Container Types](#standard-container-types)
+> [!NOTE]
+> 1. You must specify the purpose of the Container Type you are creating at creation time. Dependent on the purpose, you may or may not need to provide your Azure Subscription ID. A Container Type set for trial purpose cannot be converted for production; or vice versa.
+> 2. You must use the latest version of SharePoint Powershell for container type configurations. 
 
```

Comment:
Container Type Configuration section:
-describe the process of checking and updating container configurations

Reply:
I would do purpose first (trial and standard) and the configuration (which applies to either). The suggestion is that having one section with all configuration available would aid developers to discover features that they can enable/disable or configure in general. Each of those could link to their specific articles.